skycoin/skywire
# Add Discord start/stop logs

Did you run `make format && make check`? Yes

Part of https://github.com/SkycoinPro/skywire-services/issues/245

Depends on https://github.com/skycoin/dmsg/pull/15

How to test this PR:
- Checkout https://github.com/skycoin/dmsg against https://github.com/skycoin/dmsg/pull/15
- Start any binary providing Discord webhook URL
- Check if start/stop logs appear in Discord
